c/expr/reduceop.cc

```diff
@@ -212,6 +232,22 @@ static gmapperfn resolve0(int opcode, SType stype) {
       default:             return nullptr;
     }
   }
+
+  if (opcode == OpCode::Count) {
+    switch (stype) {
+      case ST_BOOLEAN_I1:
+      case ST_INTEGER_I1:  		return count_skipna<int8_t, int64_t>;
+      case ST_INTEGER_I2:  		return count_skipna<int16_t, int64_t>;
+      case ST_INTEGER_I4:  		return count_skipna<int32_t, int64_t>;
+      case ST_INTEGER_I8: 		return count_skipna<int64_t, int64_t>;
+      case ST_REAL_F4:     		return count_skipna<float, int64_t>;
+      case ST_REAL_F8:     		return count_skipna<double, int64_t>;
+      case ST_STRING_I4_VCHAR:  return count_skipna<int32_t, int64_t>;
+      case ST_STRING_I8_VCHAR:  return count_skipna<int64_t, int64_t>;
```

Comment:
these should be `<uint32_t, ...>` and `<uint64_t, ...>`.

Reply:
Yes, you're right. Will fix it. Thanks.

Reply:
Fixed.